test/mocks/SemanticjQuery-mock.js

```diff
@@ -20,6 +20,7 @@ jQuery.ajax = sandbox.stub().returnsThis();
 const jQueryPlugins = {
   checkbox: sandbox.stub().returnsThis(),
   dropdown: sandbox.stub().returnsThis(),
+  form: sandbox.stub().returnsThis(),
```

Comment:
Add the form plugin to the mocked jQuery plugin list.  This is used automatically by the conformance tests.